christfellowshipchurch/web-app
# [Events] Use old layout for non-live events and Add chat watcher count

# Related Issue
- [**🏕 [Events] Remove schedule from Live event layout**](https://3.basecamp.com/3926363/buckets/17566548/todos/3094801989)
- [**🏕 [Events] Use existing layout for non-live events**](https://3.basecamp.com/3926363/buckets/17566548/todos/3076462962)
- [**🏕 [Chat] Show watcher count**](https://3.basecamp.com/3926363/buckets/17566548/todos/3100735047)

# Changes or Updates
- Cleans up component structure so the old Events layout is rendered with the new components for non-live events
- Pulls **Schedule** info out of side-panel, and puts it down by the description (for live events)
- Adds number of users currently watching

# Test Instructions
- Open an event that is not live locally _and_ on production site
  - I.e. http://local.christfellowship.church:3000/events/tune-in-online and https://christfellowship.church/events/tune-in-online
- 🔸 Verify the layouts closely match, across screen sizes (note: some typography tweaks were made within Schedule display)
- Open an event that is live
- 🔸 Verify:
  - The **Schedule** info is no longer in the side panel, but down by the description area
  - The number of users currently watching is displayed in the top right... log in as a different user in an Incognito window and refresh them to validate decrement/increment as connections change
  - Gut-check responsive layout sizes

# Screenshots

|Non-Live Event|Live Event|
|---|---|
|![2020-10-09 10 54 41](https://user-images.githubusercontent.com/1812955/95602112-73a08e80-0a22-11eb-9450-d4fe5d3ce2b5.gif)|![2020-10-09 11 40 17](https://user-images.githubusercontent.com/1812955/95603476-55d42900-0a24-11eb-9089-91a8be82a30c.gif)|
|![CleanShot 2020-10-09 at 10 54 20@2x](https://user-images.githubusercontent.com/1812955/95603446-49e86700-0a24-11eb-9fa0-aba2f6552198.jpg)|![CleanShot 2020-10-09 at 11 39 31@2x](https://user-images.githubusercontent.com/1812955/95603470-52d93880-0a24-11eb-983b-23d3932e27e8.jpg)|

|Watcher Count (recorded before design polish)|
|---|
|<img src="https://user-images.githubusercontent.com/1812955/95603499-5cfb3700-0a24-11eb-9498-d2c5bda1a82c.gif" width="480" />|



# Tests
